app/services/people/crud.rb

```diff
@@ -24,12 +24,30 @@ def assessment_answers
     end
 
     def profile_identifiers
-      value = params[:attributes][:identifiers]
-      value ? { profile_identifiers: value } : {}
+      identifiers ? { profile_identifiers: identifiers } : {}
+    end
+
+    def person_identifiers
+      if identifiers
+        attributes = identifiers.each_with_object({}) do |identifier, acc|
+          acc[identifier[:identifier_type].to_sym] = identifier[:value]
+        end
+
+        attributes.slice(*Person::IDENTIFIER_TYPES)
+      else
+        {}
+      end
     end
 
     def profile_params
       params[:attributes].slice(*ATTRIBUTES)
     end
+    alias_method :person_params, :profile_params
```

Comment:
is this a good idea to alias the two? we may get confused later down the line

Reply:
We plan to separate them later. Watch this space :)